docs/devices.md

```diff
@@ -151,6 +152,7 @@ Mouse-Keyboard combos:
 [K700]: http://logitech.com/product/wireless-desktop-mk710
 [K750]: http://logitech.com/product/k750-keyboard
 [K800]: http://logitech.com/product/wireless-illuminated-keyboard-k800
+[K830]: http://www.logitech.com/en-us/product/living-room-keyboard-k830
```

Comment:
Just for standards sake, this URL also works: http://logitech.com/product/living-room-keyboard-k830